ci/bullseye_generate_report_node.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -ex
+
+sudo mkdir -p "$DAOS_BASE"
+sudo mount -t nfs "$HOSTNAME":"$HOSTPWD" "$DAOS_BASE"
+
+set +x
+if [ -n "$BULLSEYE" ]; then
+  pushd "$DAOS_BASE/bullseye"
+    sudo ./install --quiet --key "${BULLSEYE}" \
+                   --prefix /opt/BullseyeCoverage
+  popd
+  rm -rf bullseye
+  export COVFILE="$DAOS_BASE/test.cov"
+  export PATH="/opt/BullseyeCoverage/bin:$PATH"
+fi
+
+mv "$DAOS_BASE/test.cov_1" "$COVFILE"
+if [ -e "$DAOS_BASE/test.cov_2" ]; then
+  covmerge --no-banner --file "$COVFILE" "$DAOS_BASE"/test.cov_*
+fi
+
+cd "$DAOS_BASE"
+
+if [ -n "$BULLSEYE" ]; then
+  ls -l "$COVFILE" || true
```

Comment:
Debug that needs removing?

Reply:
I think it should be left in for now.  The coverage reports are going to be changing a bit after this PR is landed.

Reply:
But what value is `ls -l`ing the file adding?

Reply:
It is a diagnostic if something went wrong with a previous stage where the file did not get added to the stash.